generator/mavgen_objc.py

```diff
@@ -45,7 +45,7 @@ def generate_mavlink(directory, xml):
  @param mavlink The MVMavlink object calling this method
  @param data NSData object containing the bytes to be sent
  */
-- (BOOL)mavlink:(MVMavlink *)mavlink shouldWriteData:(NSData *)data;
+- (MAV_BOOL)mavlink:(MVMavlink *)mavlink shouldWriteData:(NSData *)data;
```

Comment:
So this this already support the BOOL enum we added? My assumption is that it did and this is why you need to change it? That was a quick addition.